sqlserver/tests/test_sqlserver.py

```diff
@@ -142,8 +142,8 @@ def test_check_adoprovider(aggregator, init_config, instance_sql2017, adoprovide
 
 
 @pytest.mark.e2e
-def test_check_docker_e2e(dd_agent_check, init_config, instance_docker):
-    aggregator = dd_agent_check({'init_config': init_config, 'instances': [instance_docker]}, rate=True)
+def test_check_docker_e2e(dd_agent_check, init_config, instance_e2e):
+    aggregator = dd_agent_check({'init_config': init_config, 'instances': [instance_e2e]}, rate=True)
```

Comment:
not: I think you can simplify this because you already yield the correct configuration in dd_environment

```suggestion
def test_check_docker_e2e(dd_agent_check):
    aggregator = dd_agent_check(rate=True)
```

Reply:
I tend to favour the current explicit way though.

Reply:
I understand the argument, but isn't that the only point of yielding the config in the dd_environment fixture ? 

Passing those args here just means that there are a few extra steps: copying the config to the container and running the check command with "-c".

Reply:
> I understand the argument, but isn't that the only point of yielding the config in the dd_environment fixture ?

I see it as a config useful for running `ddev env check` and less for e2e tests.

> Passing those args here just means that there are a few extra steps: copying the config to the container and running the check command with "-c".

It's more steps, but I think the time needed for this extra step is negligible.

Reply:
👍